logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationGroupRepository.kt

```diff
@@ -145,7 +145,7 @@ internal class ConversationGroupRepositoryImpl(
             )
         }.onSuccess { response ->
             if (response is ConversationMemberAddedResponse.Changed) {
-                memberJoinEventHandler.handle(eventMapper.conversationMemberJoin("", response.event))
+                memberJoinEventHandler.handle(eventMapper.conversationMemberJoin("", response.event, true))
```

Comment:
Hmm just thinking out loud. I think it's fine to consider them transient and not update the last  processed event ID  here because we will get the same events on the next quick sync and the processed event ID will be updated then